exporter/googlemanagedprometheusexporter/README.md

```diff
@@ -182,3 +182,127 @@ processors:
     - cluster
     - location
 ```
+
+## Feature-gates
+
+- `exporter.googlemanagedpromethues.intToDouble`: Change all metric datapoint type to double
```

Comment:
Would be good to describe if the feature gate is enabled or not by default. Also does this gate have a schedule when it's expected to switch to enabled by default or when it's going to be stabilized, removed?

See here for an example: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/hostmetricsreceiver/README.md#feature-gates

Reply:
I'm honestly not sure. Enabling it by default will likely break a lot of users when they upgrade.  We might need to implement the automatic deletion of "bad" metric descriptors ourselves...

Reply:
documented the default.